src/components/timeline/examples/timeline/index.njk

```diff
@@ -0,0 +1,28 @@
+{% from "components/timeline/_macro.njk" import onsTimeline %}
+
+{{
+    onsTimeline({
+    "items": [
+            {
+                "heading": 'September to October 2020',
+                "content": '<ul class=\"list--bare\"><li class=\"list__item\">Meet your Community Engagement Manager and set up regular catch up meetings</li><li class=\"list__item\">Local Authority Toolkit available online</li></ul>'
```

Comment:
I think we would want to reuse the list component here. Other than that looks good

Reply:
I have updated the timeline to use the onsList as optional. Thanks.